src/main/resources/assets/colossalchests/lang/en_US.lang

```diff
@@ -26,3 +26,33 @@ multiblock.colossalchests.error.prefix=ColossalChests
 multiblock.colossalchests.error.prefix.info=Colossal Chest multiblock structure error
 multiblock.colossalchests.error.unexpected=Unexpected Colossal Chest structure error.
 multiblock.colossalchests.error.material=Found a block of material %s at %s, while %s was expected.
+
+# Configuration
+config.colossalchests=%s configuration
+
+## Config categories
+config.colossalchests.block=Block settings
+config.colossalchests.general=General settings
+config.colossalchests.machine=Machine settings
+config.colossalchests.core=Core settings
+config.colossalchests.modcompat=Mod compatibility settings
+
+## General settings
+config.colossalchests.colossalChest.chestAnimation=Chest animation
+config.colossalchests.general.ejectItemsOnDestroy=Eject items on destroy
+config.colossalchests.general.metalVariants=Metal variants
+
+## Machine settings
+config.colossalchests.colossalChest.maxSize=Maximal size of colossal chest
```

Comment:
Didn't see this before, but I would make this "Maximum colossal chest diameter".